packages/neuron-wallet/tests/utils/worker.test.ts

```diff
@@ -0,0 +1,47 @@
+import path from 'path'
+import { expose, spawn, terminate } from '../../src/utils/worker'
+import { fork } from 'child_process'
+
+describe('workers', () => {
+  const noop = (..._: any) => undefined
+  const delay = (ms: number) => new Promise(resolve => setTimeout(resolve, ms))
+
+  it('expose should send object keys as channels to the master process', async () => {
```

Comment:
I would suggest using the pattern similar to the other tests in the sync queue module:
```js
describe('#function', () => {
  describe('condition', () => {
    it('success case')
    it('failed case')
  })
})
```

Reply:
Fixed.